packages/hint-validate-summary/README.md

```diff
@@ -0,0 +1,62 @@
+# validate-summary (`@hint/hint-validate-summary`)
+
+Summary tag should have display: list-item only.
+
+## Why is this important?
+
+Explain why this hint is important for your users
```

Comment:
We should fill out the rest of this documentation as well. Key things to capture are that changing `display` on a `<summary>` tag to anything other than `list-item` will cause the open/close icon to disappear in browsers which implement the latest version of the standard (e.g. Firefox).

Also good to include supporting links if possible (which I think were included in the original issue).